main.js

```diff
@@ -92,41 +92,48 @@ function removeError(){
 
 // Creating html registration form
 function createLoginForm() {
-    const div_form_container = document.createElement("div");
-    div_form_container.setAttribute("class", "form_container")
-
-    const form = div_form_container.appendChild(document.createElement("form"));
-    form.setAttribute("class", "form");
-    form.setAttribute("id", "login")
-
-    const title = form.appendChild(document.createElement("h1"));
-    title.textContent = "Login form";
+    createFormContainer();
 
-    const div_form_userName = form.appendChild(document.createElement("div"));
-    div_form_userName.setAttribute("class", "form_input");
-    div_form_userName.setAttribute("id", "form_username");
-    div_form_userName.innerHTML = '<label for="username">Username:</label>' +
-    '<input type="text" name="username" id="username" placeholder="username">';
+    const parentElement = document.getElementById("login");
+    createTitle(parentElement, "Login form");
+    createInput(parentElement);
+    createButton(parentElement); 
+}
 
-    const div_form_password = form.appendChild(document.createElement("div"));
-    div_form_password.setAttribute("class", "form_input");
-    div_form_password.setAttribute("id", "form_password");
-    div_form_password.innerHTML = '<label for="password">Password:</label>' +
-    '<input type="password" name="password" id="password" placeholder="password">';
+function createFormContainer(){
```

Comment:
rework this function according to createBaseElement

Reply:
Done